config/app/com/microsoft/azure/iotsolutions/uiconfig/services/IStatusService.java

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+package com.microsoft.azure.iotsolutions.uiconfig.services;
+
+import com.google.inject.ImplementedBy;
+import com.microsoft.azure.iotsolutions.uiconfig.services.models.StatusServiceModel;
+
+@ImplementedBy(StatusService.class)
```

Comment:
nit: add some java doc for this interface to explain what is the purpose.

Reply:
done